Controller/InputProcessor.cs

```diff
@@ -1,7 +1,67 @@
+using HashPhotoSlideshow.Model;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
 namespace HashPhotoSlideshow.Controller
 {
     public class InputProcessor
     {
-        //first commit
+        public List<Photo> _photo;
```

Comment:
No need for member variables. Create a static function that takes in a path and returns a PhotoCollection. Right now there is no way for a user to select an input file? Do you expect to hardcode a path to the input file?